# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xfsprogs/asm-types.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Of course on some arches long long is not 64bit and differs from the kernel
includes, include later on.

  - Rene Rebe <rene@exactcode.de>

--- xfsprogs-2.7.11/include/platform_defs.h.in.vanilla	2006-03-29 10:29:54.000000000 +0200
+++ xfsprogs-2.7.11/include/platform_defs.h.in	2006-03-29 10:31:38.000000000 +0200
@@ -30,7 +30,9 @@
 #include <string.h>
 #include <unistd.h>
 #include <sys/types.h>
+#include <asm/types.h>
 
+#if 0
 typedef unsigned char		__u8;
 typedef signed char		__s8;
 typedef unsigned short		__u16;
@@ -39,6 +41,7 @@
 typedef signed int		__s32;
 typedef unsigned long long int	__u64;
 typedef signed long long int	__s64;
+#endif
 
 typedef __u16			__be16;
 typedef __u32			__be32;
